### PR TITLE
[Shop] Applying and Removing Promotion Coupon

### PR DESCRIPTION
**Pull request by mpysiak**
| Q               | A
|-----------------|-----
| Branch?         | bootstrap-shop
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | n/a
| License         | MIT

Just enabled and fixed behat scenarios

### CODE REVIEW

File: features/shop/promotion/applying_promotion_coupon/removing_promotion_coupon.feature [hunk 1]
```diff
@@ -12,7 +12,7 @@ Feature: Removing promotion coupon
         And I have product "PHP T-Shirt" in the cart
         And this cart has promotion applied with coupon "SANTA2016"
 
-    @todo @api
+    @api @no-ui
```
Comment (GSadee) [inline features/shop/promotion/applying_promotion_coupon/removing_promotion_coupon.feature]: Shouldn't it also be possible in the UI context? 🤔

**Review by GSadee — COMMENTED** [review]
And the rebase is needed

**Review by Wojdylak — COMMENTED** [review]
Why do we need an "apply coupon" button if the coupon field is `live`? IMO we should disable the coupon field as `no-live`.

**Comment by mpysiak** [comment]
> Why do we need an "apply coupon" button if the coupon field is `live`? IMO we should disable the coupon field as `no-live`.

Consulted with @CoderMaggie. The coupon field is now "no-live". The UX of adding/removing coupon code is like in 1.13

**Review by Wojdylak — APPROVED** [review]

File: features/shop/promotion/applying_promotion_coupon/applying_promotion_coupon.feature [hunk 2]
```diff
@@ -10,7 +10,7 @@ Feature: Applying promotion coupon
         And the store has promotion "Christmas sale" with coupon "SANTA2016"
         And this promotion gives "$10.00" discount to every order
 
-    @todo @ui @api
+    @api @ui @mink:chromedriver
```
Comment (GSadee) [inline features/shop/promotion/applying_promotion_coupon/applying_promotion_coupon.feature:13]: According to https://github.com/Sylius/Sylius/pull/16820#issuecomment-2326095118, is there a need to tag these scenarios with `@mink:chromedriver`?

Comment (mpysiak) [reply features/shop/promotion/applying_promotion_coupon/applying_promotion_coupon.feature:13]: Yeah, the coupon field is no-live. But still it is inside live component and it is submitted with button (via javascript under the hood). So, yes unfortunately it needs to be tested with JS 😞

**Review by GSadee — APPROVED** [review]

**Pull request merged**